code/game/objects/items/weapons/implants/implant.dm

```diff
@@ -520,6 +529,56 @@ the implant may become unstable and either pre-maturely inject the subject or si
 /obj/item/weapon/implant/compressed/islegal()
 	return 0
 
+
+//Cortical Implant
 /obj/item/weapon/implant/cortical
 	name = "cortical stack"
 	desc = "A fist-sized mass of biocircuits and chips."
+
+
+//Pax(Peace) Implant
+/obj/item/weapon/implant/peace
+	name = "pax implant"
+	desc = "A bean-shaped implant with a single embossed word - PAX - on it."
+	var/imp_alive = 0
+	var/msg_debounce = 0
+	var/imp_data = {"
+<b>Implant Specifications:</b><BR>
+<b>Name:</b> Pax Implant<BR>
+<b>Manufacturer:</b> Ouroboros Medical<BR>
+<b>Effect:</b> Makes the host incapable of committing violent acts.
+<b>Important Notes:</b> Effect accomplished via a chemical secreted by the implant. This chemical is neutralized by anti-toxin.<BR>
+<b>Life:</b> Sustained as long as it remains within a host. Survives on the host's nutrition. Dies upon removal.<BR>
+"}
+
+/obj/item/weapon/implant/peace/process()
+	var/mob/living/carbon/host = imp_in
+	if (isnull(host))
+		if (imp_alive == 1)
+			Destroy()
```

Comment:
`Destroy` is supposed to be called by the garbage collector, you don't want to call it manually. Instead, you want to `qdel(src)` if your intent is for the implant to delete itself, or just have it remove itself from `processing_list` if that's what you wanted.